### ISSUE TITLE
Add code of conduct

### ISSUE THREAD

**Issue by DavidWiseman** [trimble-oss/dba-dash#70]
We can probably make use of one of the templates.
https://github.com/trimble-oss/dba-dash/community/code-of-conduct/new?template=contributor-covenant

**Comment by bassackwords** [comment]
Is this actually David Wiseman?

I work at Trimble Viewpoint, we're about to start using DBA Dash so I
wanted to check it out a bit.

**Comment by DavidWiseman** [comment]
@bassackwords Yeah it's me. 😀 DBA Dash is used in a few places across Trimble now.  Feel free to reach out if you have any questions - slack, email or we can arrange meeting (I'm UK based for timezones).

**Comment by bassackwords** [comment]
Very cool.

I just realized I signed up with my personal email but I should have just
used Okta. I'm signing up over there now, username is ScrumDiddly over
there.

We're going through some very basic training tomorrow I guess for the
Viewpoint Cloud Support team, I'm excited because we struggle so bad with
performance cases and this is going to be fantastic.

I appreciate what you've made here, very impressed and thankful.

I'll reach out if I need anything.

Thank you,
